usb-power-profiling.js

```diff
@@ -1128,6 +1128,14 @@ async function startSampling() {
   }
 }
 
+async function stopSampling() {
+  if (gDevices.length == 0) {
+    console.log("No device found");
+  } else {
+    await Promise.all(gDevices.map(d => d.stopSampling()));
```

Comment:
Instead of duplicating this line of code, could you make the existing code in `process.on('SIGINT', async function() {` call this new `stopSampling` function?